PetProject/Services/CheapSharkService/CheapSharkService.swift

```diff
@@ -5,121 +5,107 @@
 //  Created by Тимофей Лукашевич on 29.03.22.
 //
 
-import Foundation
 import Combine
 import UIKit
 
 final class CheapSharkService {
     
     private let apiURL = URL(string: "https://www.cheapshark.com/api/1.0")
     private var hostURL =  URL(string: "https://www.cheapshark.com")
-    private var cancellables = Set<AnyCancellable>()
     private let networkService: NetworkService
-    let maximumDealsCount = 999
     let paginationDealsCount = 40
     
     init(networkService: NetworkService) {
         self.networkService = networkService
     }
     
-    func getStoresList(_ completion: @escaping (Result<[StoreModel], CheapSharkServiceError>) -> Void ) {
+    typealias GetStoresListPublisher = Publishers.Map<NetworkService.GetResponsePublisher, Result<[StoreModel], CheapSharkServiceError>>
```

Comment:
Try using `AnyPublisher` here instead of specifying a particular publisher type.

Reply:
Done